contributors/design-proposals/node/node-fence.md

```diff
@@ -0,0 +1,528 @@
+# Node fencing
+Status: Pending
+
+Version: Alpha
+
+Implementation Owner: Yaniv Bronhaim (ybronhei@redhat.com)
+
+## Overview
+Rationale and architecture for the addition of a fencing mechanism in Kubernetes clusters.
```

Comment:
Please reference the pod safety design and describe how this alters it (probably in scope)